src/music_album.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'item'
+
+class MusicAlbum < Item
+  attr_reader :on_spotify
+
+  def initialize(publish_date, archived, on_spotify)
+    super(publish_date, archived)
+    @on_spotify = on_spotify
+  end
+
+  private
+
+  def can_be_archived?
+    if @archived && @on_spotify
+      true
+    else
+      false
+    end
+  end
```

Comment:
According to Ruby best practices; if there is one condition only we should use shorthand 
`def can_be_archived?`
`   @archived && @on_spotify ? true : false`
` end`
On the other hand, returning true and false is unnecessary as the statement will return true and false by default.
 In that case, you can use `@archived && @on_spotify` only which will be suffecient.